test-integ/peering_commontopo/ac5_2_pq_failover_test.go

```diff
@@ -290,7 +290,7 @@ func (s *ac5_2PQFailoverSuite) testPreparedQuerySingleFailover(t *testing.T, ct
 
 			require.Equal(r, 1, queryResult.Failovers, "expected 1 prepared query failover")
 			require.Equal(r, peerClu.Name, queryResult.Nodes[0].Node.Datacenter, fmt.Sprintf("the pq results should originate from peer clu %s", peerClu.Name))
-			require.Equal(r, pqFailoverTargets[0].Peer, queryResult.Nodes[0].Checks[0].PeerName,
```

Comment:
@NiniOak could you weigh in on whether this change is still correct? I had to switch it because with the node being agentless, we don't have checks

Reply:
Never mind, I understand we shouldn't do agentless for this test